### PR TITLE
fix: initialize call providers & buttons without dynamic containers - EXO-67648

### PR DESCRIPTION
**Pull request by ahamdi**

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "babel-core": "^6.26.3",
     "babel-loader": "^7.1.4",
     "babel-preset-es2015": "^6.24.1",
+    "chokidar": "^3.5.3",
```
Comment (rdenarie) [inline webapp/package.json:20]: What is this library for ?

File: webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/rest/RESTWebRTCService.java [hunk 2]
```diff
@@ -188,7 +188,7 @@ public Response getSettings(@Context UriInfo uriInfo) {
   }
 
   @GET
-  @RolesAllowed("administrators")
+  @RolesAllowed("users")
```
Comment (rdenarie) [inline webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/rest/RESTWebRTCService.java]: Could you explain why this service must be readable by users ?
As it exposes some informations like internal urls or credentials, I prefer that it stay limited to administrators

File: webapp/package.json (hunk 1 above)
Comment (ahamdi) [reply webapp/package.json:20]: @rdenarie It is used for watching the changes on Vue files. it was missing and I was asked to install it when running ```npm run watch```

File: webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/rest/RESTWebRTCService.java (hunk 2 above)
Comment (ahamdi) [reply webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/rest/RESTWebRTCService.java]: I will revert this as I won't use the Rest endpoint. But those same settings are exposed by the JS initiation of web conferencing connector and could be gotten from there too like in https://github.com/exoplatform/web-conferencing/blob/develop/webrtc/webapp/src/main/java/org/exoplatform/webconferencing/webrtc/portlet/WebrtcProviderPortlet.java#L66
We may consider to improve this later WDYT ?

Comment (rdenarie) [reply webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/rest/RESTWebRTCService.java]: I dont know if a simple user have know it or not.
In fact, as we will remove webrtc, and keep only jitsi, it is not huge priority.
So, please create a task somewhere to check it for jitsi, and yes we will consider it later

**Review by rdenarie — APPROVED** [review]

**Pull request merged**